[UI] Add Formatted Kubernetes metadata to connections

aabidsofi19 opened the pull request:
**Notes for Reviewers**

This PR fixes #
- Format the kubernetes in the same manner as in environment page 
- can potentially remove the  environments tab now 


**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

aabidsofi19 commented:
NOTE : i have updated prettier to use "auto" for linefeed , it was giving carriage return errors on linux

theBeginner86 commented:
Moved to [useKuberneteshook](https://github.com/meshery/meshery/blob/master/ui/components/hooks/useKubernetesHook.js#L7)

theBeginner86 commented on ui/.prettierrc.yml:
```diff
@@ -10,4 +10,4 @@ tabWidth: 2
 printWidth: 100
 useTabs: false
-endOfLine: 'lf'
+endOfLine: 'crlf'
```
we should confirm this change.
It did updated all the files when I committed changes in your PR. 😓 

```
meshery git:(kubernetes-metadata) ✗ git status
On branch kubernetes-metadata
Your branch is ahead of 'aabidsofi19/kubernetes-metadata' by 1 commit.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   ui/.lintstagedrc.js
	modified:   ui/README.md
	modified:   ui/api/connections.js
	modified:   ui/api/index.js
	modified:   ui/api/meshmodel.js
	modified:   ui/api/patterns.js
	modified:   ui/components/BBChart.js
	modified:   ui/components/CatalogFilter.js
	modified:   ui/components/ConfirmationModal.js
	modified:   ui/components/Connect/connections/helm-connect/StepperContent.js
	modified:   ui/components/Connect/connections/helm-connect/constants.js
	modified:   ui/components/Connect/connections/helm-connect/utils.js
	modified:   ui/components/Connect/index.js
	modified:   ui/components/Connect/stepper/Stepper.js
	modified:   ui/components/Connect/stepper/StepperContentWrapper.js
	modified:   ui/components/Connect/stepper/index.js
	modified:   ui/components/Connect/styles.js
	modified:   ui/components/ConnectionWizard/ConfigComponents/Kubernetes.js
	modified:   ui/components/ConnectionWizard/ConfigComponents/Metrics.js
	modified:   ui/components/ConnectionWizard/ConfigComponents/ServiceMesh.js
	modified:   ui/components/ConnectionWizard/DataPanels/AdapterChip.js
	modified:   ui/components/ConnectionWizard/DataPanels/Kubernetes.js
	modified:   ui/components/ConnectionWizard/DataPanels/MesheryOperator.js
	modified:   ui/components/ConnectionWizard/DataPanels/Metrics.js
	modified:   ui/components/ConnectionWizard/DataPanels/ServiceMesh.js
	modified:   ui/components/ConnectionWizard/Screens/CompletionScreen.js
	modified:   ui/components/ConnectionWizard/Screens/KubernetesScreen.js
	modified:   ui/components/ConnectionWizard/Screens/MesheryOperatorScreen.js
	modified:   ui/components/ConnectionWizard/Screens/MetricsScreen.js
	modified:   ui/components/ConnectionWizard/Screens/ServiceMeshScreen.js
	modified:   ui/components/ConnectionWizard/ScrollIndicator.js
	modified:   ui/components/ConnectionWizard/ServiceCard.js
	modified:   ui/components/ConnectionWizard/Stepper.js
	modified:   ui/components/ConnectionWizard/helpers/common.js
	modified:   ui/components/ConnectionWizard/helpers/kubernetesHelpers.js
	modified:   ui/components/ConnectionWizard/helpers/mesheryOperator.js
	modified:   ui/components/ConnectionWizard/helpers/metrics.js
	modified:   ui/components/ConnectionWizard/helpers/serviceMeshes.js
	modified:   ui/components/ConnectionWizard/icons/GrafanaIcon.js
	modified:   ui/components/ConnectionWizard/icons/KubernetesIcon.js
	modified:   ui/components/ConnectionWizard/icons/LinkerdIcon.js
	modified:   ui/components/ConnectionWizard/icons/MesheryOperatorIcon.js
	modified:   ui/components/ConnectionWizard/icons/OpenServiceMeshIcon.js
	modified:   ui/components/ConnectionWizard/icons/PrometheusIcon.js
	modified:   ui/components/ConnectionWizard/icons/apps.js
	modified:   ui/components/ConnectionWizard/icons/icons.js
	modified:   ui/components/ConnectionWizard/index.js
	modified:   ui/components/CustomAvatar.js
	modified:   ui/components/CustomTableFooter.js
	modified:   ui/components/CustomToolbarSelect.js
	modified:   ui/components/Dashboard/ConnectionCharts.js
	modified:   ui/components/Dashboard/DashboardMeshModelGraph.js
	modified:   ui/components/Dashboard/WorkloadChart.js
	modified:   ui/components/DashboardComponent.js
	modified:   ui/components/DataFormatter/README.md
	modified:   ui/components/DataFormatter/index.js
	modified:   ui/components/DatabaseSummary.js
	modified:   ui/components/DryRun/DryRunComponent.js
	modified:   ui/components/DuplicatesDataTable.js
	modified:   ui/components/EmptyState/K8sContextEmptyState.js
	modified:   ui/components/ErrorBoundary.js
	modified:   ui/components/ErrorHandling.js
	modified:   ui/components/ErrorPage.js
	modified:   ui/components/ExpansionPanels.js
	modified:   ui/components/ExtensionSandbox.js
	modified:   ui/components/Filters.js
	modified:   ui/components/FlipCard.js
	modified:   ui/components/General/ConnectClustersBtn.js
	modified:   ui/components/General/ErrorBoundary.js
	modified:   ui/components/General/TipsCarousel.js
	modified:   ui/components/GenericModal.js
	modified:   ui/components/Header.js
	modified:   ui/components/LoadingComponents/Animations/AnimatedFilter.js
	modified:   ui/components/LoadingComponents/Animations/AnimatedLightMeshery.js
	modified:   ui/components/LoadingComponents/Animations/AnimatedMeshPattern.js
	modified:   ui/components/LoadingComponents/Animations/AnimatedMeshSync.js
	modified:   ui/components/LoadingComponents/Animations/AnimatedMeshery.js
	modified:   ui/components/LoadingComponents/LoadingComponent.js
	modified:   ui/components/MeshAdapterConfigComponent.js
	modified:   ui/components/MeshConfigComponent.js
	modified:   ui/components/MeshModelRegistry/MeshModelComponent.js
	modified:   ui/components/MeshModelRegistry/MeshModelDetails.js
	modified:   ui/components/MeshModelRegistry/MesheryTreeView.js
	modified:   ui/components/MesheryAdapterPlayComponent.js
	modified:   ui/components/MesheryApplications.js
	modified:   ui/components/MesheryApplications/ApplicationsCard.js
	modified:   ui/components/MesheryApplications/ApplicationsGrid.js
	modified:   ui/components/MesheryChart.js
	modified:   ui/components/MesheryChartDialog.js
	modified:   ui/components/MesheryCredentialComponent.js
	modified:   ui/components/MesheryDateTimePicker.js
	modified:   ui/components/MesheryFilters/FiltersCard.js
	modified:   ui/components/MesheryFilters/FiltersGrid.js
	modified:   ui/components/MesheryMeshInterface/PatternService/CustomTextTooltip.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSF.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/Accordion.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/ArrayFieldTemlate.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomBaseInput.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomCheckboxWidget.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomDateTimeWidget.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomFileWidget.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomSelectWidget.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomTextAreaWidget.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomTextWidget.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/CustomURLWidget.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/FieldTemplate.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/ObjectFieldTemplate.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/WrapIfAdditionalTemplate.js
	modified:   ui/components/MesheryMeshInterface/PatternService/RJSF_wrapper.js
	modified:   ui/components/MesheryMeshInterface/PatternService/helper.js
	modified:   ui/components/MesheryMeshInterface/PatternService/index.js
	modified:   ui/components/MesheryMeshInterface/PatternServiceForm.js
	modified:   ui/components/MesheryMeshInterface/PatternServiceFormCore.js
	modified:   ui/components/MesheryMeshInterface/helpers.js
	modified:   ui/components/MesheryMetrics.js
	modified:   ui/components/MesheryPatterns.js
	modified:   ui/components/MesheryPatterns/Cards.styles.js
	modified:   ui/components/MesheryPatterns/CustomToolbarSelect.js
	modified:   ui/components/MesheryPatterns/Grid.styles.js
	modified:   ui/components/MesheryPatterns/MesheryPatternCard.js
	modified:   ui/components/MesheryPatterns/MesheryPatternGridView.js
	modified:   ui/components/MesheryPerformance/Dashboard.js
	modified:   ui/components/MesheryPerformance/PerformanceCalendar.js
	modified:   ui/components/MesheryPerformance/PerformanceCard.js
	modified:   ui/components/MesheryPerformance/PerformanceProfileGrid.js
	modified:   ui/components/MesheryPerformance/PerformanceProfiles.js
	modified:   ui/components/MesheryPerformance/PerformanceResults.js
	modified:   ui/components/MesheryPerformance/assets/facebookIcon.js
	modified:   ui/components/MesheryPerformance/assets/linkedinIcon.js
	modified:   ui/components/MesheryPerformance/assets/twitterIcon.js
	modified:   ui/components/MesheryPerformance/helper.js
	modified:   ui/components/MesheryPerformance/index.js
	modified:   ui/components/MesheryPlayComponent.js
	modified:   ui/components/MesheryProgressBar.js
	modified:   ui/components/MesheryResultDialog.js
	modified:   ui/components/MesheryResults.js
	modified:   ui/components/MesherySMIResults.js
	modified:   ui/components/MesherySettings.js
	modified:   ui/components/MesherySettingsEnvButtons.js
	modified:   ui/components/MesherySettingsPerformanceComponent.js
	modified:   ui/components/MesherySnackbarWrapper.js
	modified:   ui/components/Modal.js
	modified:   ui/components/Modals/ImportModal.js
	modified:   ui/components/Modals/Information/InfoModal.js
	modified:   ui/components/Modals/Information/styles.js
	modified:   ui/components/Modals/PublishModal.js
	modified:   ui/components/Navigator.js
	modified:   ui/components/NavigatorExtension.js
	modified:   ui/components/NodeDetails.js
	modified:   ui/components/NotificationCenter/README.md
	modified:   ui/components/NotificationCenter/constants.js
	modified:   ui/components/NotificationCenter/filter.js
	modified:   ui/components/NotificationCenter/index.js
	modified:   ui/components/NotificationCenter/metadata.js
	modified:   ui/components/NotificationCenter/notification.js
	modified:   ui/components/NotificationCenter/notificationCenter.style.js
	modified:   ui/components/PageContext.js
	modified:   ui/components/Popup.js
	modified:   ui/components/PromptComponent.js
	modified:   ui/components/ReactSelectWrapper.js
	modified:   ui/components/RemoteComponent.js
	modified:   ui/components/TroubleshootingComponent.js
	modified:   ui/components/TroubleshootingModalComponent.js
	modified:   ui/components/TypingFilter/README.md
	modified:   ui/components/TypingFilter/index.js
	modified:   ui/components/TypingFilter/state.js
	modified:   ui/components/TypingFilter/style.js
	modified:   ui/components/TypingFilter/utils.js
	modified:   ui/components/URLUploader.js
	modified:   ui/components/UploadImport.js
	modified:   ui/components/User.js
	modified:   ui/components/UserPreferences.js
	modified:   ui/components/Validation.js
	modified:   ui/components/VerticalCarousel/VerticalCarousel.js
	modified:   ui/components/ViewSwitch.js
	modified:   ui/components/YamlDialog.js
	modified:   ui/components/configuratorComponents/CodeEditor.js
	modified:   ui/components/configuratorComponents/CustomBreadCrumb.js
	modified:   ui/components/configuratorComponents/MeshModel/LazyComponentForm.js
	modified:   ui/components/configuratorComponents/MeshModel/hooks/types.js
	modified:   ui/components/configuratorComponents/MeshModel/hooks/useDesignLifecycle.js
	modified:   ui/components/configuratorComponents/MeshModel/index.js
	modified:   ui/components/configuratorComponents/MeshModel/styledComponents/AppBar.js
	modified:   ui/components/configuratorComponents/NameToIcon.js
	modified:   ui/components/connections/ConnectionChip.js
	modified:   ui/components/connections/MeshsyncTable.js
	modified:   ui/components/connections/styles.js
	modified:   ui/components/extensions/adapters/adapters.js
	modified:   ui/components/extensions/adapters/constants.js
	modified:   ui/components/extensions/adapters/index.js
	modified:   ui/components/extensions/index.js
	modified:   ui/components/graphql/mutations/AdapterStatusMutation.js
	modified:   ui/components/graphql/mutations/OperatorStatusMutation.js
	modified:   ui/components/graphql/queries/AddonsStatusQuery.js
	modified:   ui/components/graphql/queries/CatalogFilterQuery.js
	modified:   ui/components/graphql/queries/CatalogPatternQuery.js
	modified:   ui/components/graphql/queries/ControlPlanesQuery.js
	modified:   ui/components/graphql/queries/DataPlanesQuery.js
	modified:   ui/components/graphql/queries/FetchAllResultsQuery.js
	modified:   ui/components/graphql/queries/MeshModelSummaryQuery.js
	modified:   ui/components/graphql/queries/MeshsyncStatusQuery.js
	modified:   ui/components/graphql/queries/NamespaceQuery.js
	modified:   ui/components/graphql/queries/NatsStatusQuery.js
	modified:   ui/components/graphql/queries/OperatorStatusQuery.js
	modified:   ui/components/graphql/queries/PerformanceProfilesQuery.js
	modified:   ui/components/graphql/queries/PerformanceResultQuery.js
	modified:   ui/components/graphql/queries/ResetDatabaseQuery.js
	modified:   ui/components/graphql/queries/TelemetryComponentsQuery.js
	modified:   ui/components/graphql/subscriptions/ClusterResourcesSubscription.js
	modified:   ui/components/graphql/subscriptions/ConfigurationSubscription.js
	modified:   ui/components/graphql/subscriptions/EventsSubscription.js
	modified:   ui/components/graphql/subscriptions/K8sContextSubscription.js
	modified:   ui/components/graphql/subscriptions/MeshModelSummarySubscription.js
	modified:   ui/components/graphql/subscriptions/MeshSyncEventsSubscription.js
	modified:   ui/components/graphql/subscriptions/MeshSyncStatusSubscription.js
	modified:   ui/components/graphql/subscriptions/MesheryControllersStatusSubscription.js
	modified:   ui/components/graphql/subscriptions/OperatorStatusSubscription.js
	modified:   ui/components/graphql/subscriptions/PerformanceProfilesSubscription.js
	modified:   ui/components/graphql/subscriptions/PerformanceResultSubscription.js
	modified:   ui/components/hooks/useKubernetesHook.js
	modified:   ui/components/layout.js
	modified:   ui/components/load-test-timer-dialog.js
	modified:   ui/components/reusable-tooltip.js
	modified:   ui/components/schemas/credentials/grafana.js
	modified:   ui/components/schemas/credentials/kubernetes.js
	modified:   ui/components/schemas/credentials/prometheus.js
	modified:   ui/components/shapes/Octagon.js
	modified:   ui/components/shapes/RoundedTriangle.js
	modified:   ui/components/subscription/comparatorFns.js
	modified:   ui/components/subscription/helpers.js
	modified:   ui/components/subscription/mergeFns.js
	modified:   ui/components/subscription/subscriptionhandler.js
	modified:   ui/components/telemetry/grafana/GrafanaCharts.js
	modified:   ui/components/telemetry/grafana/GrafanaComponent.js
	modified:   ui/components/telemetry/grafana/GrafanaConfigComponent.js
	modified:   ui/components/telemetry/grafana/GrafanaCustomChart.js
	modified:   ui/components/telemetry/grafana/GrafanaCustomCharts.js
	modified:   ui/components/telemetry/grafana/GrafanaCustomGaugeChart.js
	modified:   ui/components/telemetry/grafana/GrafanaDateRangePicker.js
	modified:   ui/components/telemetry/grafana/GrafanaDisplaySelection.js
	modified:   ui/components/telemetry/grafana/GrafanaMetricsCompare.js
	modified:   ui/components/telemetry/grafana/GrafanaSelectionComponent.js
	modified:   ui/components/telemetry/prometheus/PrometheusComponent.js
	modified:   ui/components/telemetry/prometheus/PrometheusConfigComponent.js
	modified:   ui/components/telemetry/prometheus/PrometheusSelectionComponent.js
	modified:   ui/constants/colors.js
	modified:   ui/constants/endpoints.js
	modified:   ui/constants/navigator.js
	modified:   ui/css/disableComponent.styles.js
	modified:   ui/css/grid.style.js
	modified:   ui/css/icons.styles.js
	modified:   ui/cypress.config.js
	modified:   ui/cypress/actionHelpers/graphql-test-utils.js
	modified:   ui/cypress/actionHelpers/service-mesh-configuration-management.js
	modified:   ui/cypress/e2e/e2e/service_mesh_configuration_management_spec.js
	modified:   ui/cypress/e2e/e2e/service_mesh_lifecycle_settings_spec.js
	modified:   ui/cypress/e2e/e2e/userpreference_spec.js
	modified:   ui/cypress/e2e/integration/configuration_filters_spec.js
	modified:   ui/cypress/e2e/integration/indexui_spec.js
	modified:   ui/cypress/e2e/integration/settings_spec.js
	modified:   ui/cypress/e2e/integration/userpreference_spec.js
	modified:   ui/cypress/e2e/sample_spec.js
	modified:   ui/cypress/fixtures/clusterVersion.json
	modified:   ui/cypress/fixtures/configuration/filter-stubs/initial-filters.json
	modified:   ui/cypress/fixtures/configuration/filter-stubs/post-upload-filters.json
	modified:   ui/cypress/fixtures/example.json
	modified:   ui/cypress/fixtures/getMeshAdapters.json
	modified:   ui/cypress/fixtures/postMeshManage.json
	modified:   ui/cypress/fixtures/stats.json
	modified:   ui/cypress/fixtures/sync.json
	modified:   ui/cypress/plugins/index.js
	modified:   ui/cypress/support/commands.js
	modified:   ui/cypress/support/e2e.js
	modified:   ui/lib/chartjs-formatter.js
	modified:   ui/lib/data-fetch.js
	modified:   ui/lib/event-types.js
	modified:   ui/lib/path.js
	modified:   ui/lib/prePopulatedOptions.js
	modified:   ui/lib/relayEnvironment.js
	modified:   ui/lib/store.js
	modified:   ui/lib/trueRandom.js
	modified:   ui/nodemon.json
	modified:   ui/pages/404.js
	modified:   ui/pages/_app.js
	modified:   ui/pages/_document.js
	modified:   ui/pages/api/__coverage__.js
	modified:   ui/pages/configuration/designs/configurator.js
	modified:   ui/pages/configuration/designs/index.js
	modified:   ui/pages/configuration/filters.js
	modified:   ui/pages/connect/[...connection].js
	modified:   ui/pages/extension/AccessMesheryModal.js
	modified:   ui/pages/extension/[...component].js
	modified:   ui/pages/extensions.js
	modified:   ui/pages/index.js
	modified:   ui/pages/management/[mesh].js
	modified:   ui/pages/management/connections.js
	modified:   ui/pages/management/index.js
	modified:   ui/pages/performance/index.js
	modified:   ui/pages/performance/profiles.js
	modified:   ui/pages/performance/results.js
	modified:   ui/pages/settings.js
	modified:   ui/pages/smi_results.js
	modified:   ui/pages/system/connections.js
	modified:   ui/pages/user/preferences.js
	modified:   ui/relay.config.js
	modified:   ui/remote-component.config.js
	modified:   ui/rtk-query/connection.js
	modified:   ui/rtk-query/index.js
	modified:   ui/rtk-query/meshModel.js
	modified:   ui/rtk-query/notificationCenter.js
	modified:   ui/rtk-query/schema.js
	modified:   ui/rtk-query/user.js
	modified:   ui/store/index.js
	modified:   ui/store/slices/events.js
	modified:   ui/themes/app.js
	modified:   ui/themes/configurationTableTheme.js
	modified:   ui/themes/index.js
	modified:   ui/themes/rjsf.js
	modified:   ui/ui_dev_server.js
	modified:   ui/utils/Elements.js
	modified:   ui/utils/Enum.js
	modified:   ui/utils/ExtensionPointSchemaValidator.js
	modified:   ui/utils/PascalCaseToKebab.js
	modified:   ui/utils/TooltipButton.js
	modified:   ui/utils/URLValidator.js
	modified:   ui/utils/camelCaseToSentanceCase.js
	modified:   ui/utils/can.js
	modified:   ui/utils/charts.js
	modified:   ui/utils/context/schemaSet.js
	modified:   ui/utils/custom-column.js
	modified:   ui/utils/custom-search.js
	modified:   ui/utils/cypress-gql/graphql-test-utils.js
	modified:   ui/utils/data-table.js
	modified:   ui/utils/debounce.js
	modified:   ui/utils/dimension.js
	modified:   ui/utils/disabledComponents.js
	modified:   ui/utils/fileDownloader.js
	modified:   ui/utils/hooks/useMeshModelComponents.js
	modified:   ui/utils/hooks/useNotification.js
	modified:   ui/utils/hooks/usePreventUserFromLeavingPage.js
	modified:   ui/utils/hooks/useStateCB.js
	modified:   ui/utils/lightenOrDarkenColor.js
	modified:   ui/utils/multi-ctx.js
	modified:   ui/utils/nameMapper.js
	modified:   ui/utils/normalizeURI.js
	modified:   ui/utils/objects.js
	modified:   ui/utils/responsive-column.js
	modified:   ui/utils/theme.js
	modified:   ui/utils/utils.js
	modified:   ui/utils/versionSort.js
	modified:   ui/utils/webApis.js
	modified:   ui/utils/workloadFilter.js
	modified:   ui/utils/zIndex.js

no changes added to commit (use "git add" and/or "git commit -a")
➜  meshery git:(kubernetes-metadata) ✗
```

aabidsofi19 replied:
oh , it should have been `auto`  though i will just  revert  now

aabidsofi19 commented:
New Changes : 
-  Fixed the connection chips bug 
-  Refactored the meshery specific operations into separate hooks ( like useMesheryOperator , useMeshsyncController ) 

@theBeginner86 @leecalcote   are we good to disable the environments tab with these changes

theBeginner86 commented on ui/components/ConnectionWizard/helpers/mesheryOperator.js:
```diff
@@ -7,8 +9,20 @@ export const isMesheryOperatorConnected = ({ operatorInstalled }) => operatorIns
+    k8scontextID: id,
+  }).subscribe({
+    next: (res) => {
+      console.log('res', res);
```
```suggestion

```

theBeginner86 commented on ui/components/MeshConfigComponent.js:
```diff
@@ -311,7 +311,7 @@ function MesherySettingsNew({ classes, updateProgress, operatorState, k8sconfig
         event_type: EVENT_TYPES.SUCCESS,
       });
-
+      console.log('contextId', contextId);
```
```suggestion
```

theBeginner86 commented on ui/components/connections/index.js:
```diff
@@ -519,7 +518,7 @@ function Connections({ classes, updateProgress, /*onOpenCreateConnectionModal,*/
         const colSpan = rowData.length;
         const connection = connections && connections[tableMeta.rowIndex];
-
+        console.log('connection', connection);
```
```suggestion
```

theBeginner86 commented:
Removed debug log 👍

theBeginner86 commented:
> New Changes :
> 
> * Fixed the connection chips bug
> * Refactored the meshery specific operations into separate hooks ( like useMesheryOperator , useMeshsyncController )
> 
> @theBeginner86 @leecalcote are we good to disable the environments tab with these changes

Yes, I think we should disable it 👍

aabidsofi19 commented:
> Removed debug log 👍

Thank You
